subprojects/guide/src/asciidoc/icon-packs.adoc

```diff
@@ -478,17 +478,42 @@ compile '{project-group}:ikonli-materialdesign-pack:{project-version}'
 ----
 
 == Metrizeicons
- * Font version: {metrizeiconsVersion}
- * link:http://www.alessioatzeni.com/metrize-icons/[]
- * link:cheat-sheet-metrizeicons.html[Cheat-Sheet]
+     * Font version: {metrizeiconsVersion}
+     * link:http://www.alessioatzeni.com/metrize-icons/[]
+     * link:cheat-sheet-metrizeicons.html[Cheat-Sheet]
+
+    === Installing
+
+    [source,groovy]
+    [subs="attributes"]
+    .gradle
+    ----
+    compile '{project-group}:ikonli-metrizeicons-pack:{project-version}'
+    ----
+
+    [source,xml]
+    [subs="attributes,verbatim"]
+    .maven
+    ----
+    <dependency>
+        <groupId>{project-group}</groupId>
+        <artifactId>ikonli-metrizeicons-pack</artifactId>
+        <version>{project-version}</version>
+    </dependency>
+    ----
```

Comment:
Hmm looks like some extra formatting was added to the previous block. Could you update the file once more without these extra spaces? Thank you 😄